### ISSUE TITLE
Empty (black) page when clicking on Run Image if podman machine is stopped, also not recovering it from a crash

### ISSUE THREAD

**Issue by scscgit** [containers/podman-desktop#8449]
### Bug description

When you click on the "Run Image", if the `podman machine inspect` has `"State": "stopped"`, it shows blank like this:

![image](https://github.com/user-attachments/assets/eef50a95-39e9-46eb-bc75-a5dd986b3b38)

Additionally, there are several issues with Podman Desktop:

* It doesn't signal in any way that the machine has became on-responsive
* It doesn't suggest that the machine should be (re-)started when it's stopped

Note that a typical situation on Windows is that WSL2 often becomes unresponsive and needs to be restarted by killing `wslservice.exe` process. See many top issues like https://github.com/microsoft/WSL/issues/8529. It would be great if Podman Desktop recognized this problem and provided user-friendly solutions for automatic or manual recovery.
* One of the workarounds I'm missing in Docker Desktop is that previously running containers aren't started when the engine restarts.

### Operating system

Windows 10, WSL2

### Installation Method

Installer from website/GitHub releases

### Version

1.11.1

### Steps to reproduce

1. `podman machine stop`
2. Quickly click on "Run Image" before the image disappears

Alternatively:
1. Have podman running in WSL2 on Windows
2. `taskkill /f /im "wslservice.exe"`
3. You can take longer time to click on "Run Image", because the image won't disappear.

### Relevant log output

```shell
`log : main ↪️ event is [object Object]` is repeated over and over again. Note that logs don't include date, which I believe you could also fix.
```

### Additional context

When WSL gets killed, **Docker Desktop** displays this popup that lets us restart the engine. Similar feature is missing in **Podman Desktop**, so we may forget to run the machine altogether (when it stops for **any reason**), and there isn't even any information saying that the machine is stopped.

![image](https://github.com/user-attachments/assets/e5f7d06f-9dee-4b18-aaa3-2d8e2d221f57)

If it fails to restart, it doesn't correctly recover though, and it quits. My **suggestion** would be to definitely keep re-trying to restart, even without requiring any user interaction, at least if user chooses that behavior. Though, I noticed that I may need to kill `wslservice.exe` **again** to recover from the following error if it repeats each time:

![image](https://github.com/user-attachments/assets/9225a4f2-73d4-4cfa-86cd-45e52770996e)

**Comment by odockal** [comment]
@scscgit Hey, thanks for opening the issue. I was able to reproduce the problem, I tried to stop the machine and then quickly open "Run Image" option which led to blank screen. 
But I was able to leave the page and podman desktop functioned correctly, I am not sure about the "crash". 

Also, if you do not have any podman/ podman machine running, it usually shows this message on the ie. images page:
![Image](https://github.com/user-attachments/assets/94ccb2e3-8c3c-4435-8082-c5d70f7f1a61)

Anyway, we will look at what we can do with the bug and enhancement.

**Comment by cdrage** [comment]
Investigating this, I'm unable to reproduce this in the same way as the original issue.

https://github.com/containers/podman-desktop/pull/7975 has fixed the issue with podman machine disconnecting / not refreshing and resolved two other issues which were similar in problems.

I noticed that you are running 1.11.1 in the screenshot which is probably why it's having that error.

Can you please try again with 1.12.0 and let us know if you encounter the issue again @scscgit ?

**Comment by cdrage** [comment]
Closing this for now as I'm still unable to reproduce this on the latest main commit. The issue may have also been fixed by the commit in #7975 that landed in 1.12 of Podman Desktop.

Closing for now, please re-comment if this issue appears again.
